src/main/webapp/i18n/de/programmingExercise.json

```diff
@@ -710,6 +710,18 @@
                 "pushExercise": "Lade den Aufgaben-Code in das entsprechende Repository. Es wird in <code>{{checkoutDirectory}}</code> ausgecheckt.",
                 "pushSolution": "Lade den Lösungs-Code in das entsprechende Repository. Es wird nicht ausgecheckt.",
                 "pushTest": "Lade den Test-Code in das entsprechende Repository. Es wird in <code>{{checkoutDirectory}}</code> ausgecheckt."
+            },
+            "dockerFlags": {
+                "disableNetworkAccess": {
+                    "title": "Internetzugang beim Ausführen der Studierendeneinreichung deaktivieren",
+                    "description": "Aktiviere diese Option, um den Netzwerkzugang für Container zu deaktivieren. Das Netzwerk wird beim Ausführen der Studierendeneinreichung deaktiviert. Die Abhängigkeiten werden jedoch zuvor im Container heruntergeladen und zwischengespeichert.",
+                    "warning": "Wenn der Internetzugang deaktiviert ist, wird ein Skript ausgeführt, das Abhängigkeiten herunterlädt und im Container zwischenspeichert, bevor die Einreichung des Studierenden gebaut und getestet wird."
+                },
+                "envVars": {
+                    "title": "Umgebungsvariablen",
+                    "description": "Füge Umgebungsvariablen zum Docker-Container hinzu. Zum Beispiel: 'API_KEY'=\"ABCD 1234\",API_SECRET='XYZ',\"API_TOKEN\"=1234 (Max 1000 Zeichen)",
+                    "invalid": "Ungültiges Format der Umgebungsvariable. Schlüssel und Werte können in einfachen oder doppelten Anführungszeichen stehen. Zum Beispiel: 'API_KEY'=\"ABCD 1234\",API_SECRET='XYZ',\"API_TOKEN\"=1234"
```

Comment:
Would it make sense to restrict this a bit further? That would make the regex simpler, and doesn't really make a difference for the user imo. So only single quotes e.g.

Reply:
If only a single quote is allowed, it may be a bit confusing to enter single quotes in your environment variables.